web/portal/client/src/entities/IvrEntry/Form.tsx

```diff
@@ -29,8 +29,6 @@ const Form = (props: EntityFormProps): JSX.Element => {
         'numberValue',
         'extension',
         'voicemail',
-        'conditional',
```

Comment:
this fields are shown when routeType is conditional or voicemail. They must be handle through visual toggles

Reply:
when routeType is conditional, conditionalRoute is shown